### PR TITLE
Enable docker multiarch build

### PR DESCRIPTION
**Pull request by k0da**
Build docker image for aarch64 and combine them into a single manifest

Signed-off-by: Dinar Valeev <dinar.valeev@absa.africa>

### CODE REVIEW

**Comment by k0da** [comment]
fixes https://github.com/AbsaOSS/k8gb/issues/243

File: Makefile [hunk 1]
```diff
@@ -168,13 +168,24 @@ dns-smoke-test:
...
+	$(call docker-push-arch amd64)
+	$(call docker-push-arch arm64)
+
+# create and push docker manifest
+.PHONY: docker-manifest
+docker-manifest: docker-push
+	docker manifest create ${IMG} \
+		${IMG}-amd64 \
+		${IMG}-arm64
+	docker manifest annotate ${IMG} ${IMG}-arm64 \
```
Comment (ytsarev) [inline Makefile:186]: amd64 doesn't require annotation ?

Comment (k0da) [reply Makefile:186]: No, amd64 is a default

File: Makefile [hunk 2]
```diff
@@ -168,13 +168,24 @@ dns-smoke-test:
...
+	$(call docker-build-arch,amd64)
+	$(call docker-build-arch,arm64)
 
 # push the docker image
 .PHONY: docker-push
 docker-push:
-	docker push $(IMG)
-
+	$(call docker-push-arch amd64)
+	$(call docker-push-arch arm64)
```
Comment (ytsarev) [inline Makefile:178]:
```suggestion
	$(call docker-push-arch, arm64)
```

File: Makefile [hunk 3]
```diff
@@ -168,13 +168,24 @@ dns-smoke-test:
...
-	docker build . -t $(IMG)
+	$(call docker-build-arch,amd64)
+	$(call docker-build-arch,arm64)
 
 # push the docker image
 .PHONY: docker-push
 docker-push:
-	docker push $(IMG)
-
+	$(call docker-push-arch amd64)
```
Comment (ytsarev) [inline Makefile:177]:
```suggestion
	$(call docker-push-arch,amd64)
```

**Review by ytsarev — CHANGES_REQUESTED** [review]

**Review by ytsarev — CHANGES_REQUESTED** [review]
@k0da can we keep amd64 optional under flag? the local build of it is extremely resource-demanding

**Review by ytsarev — APPROVED** [review]

**Pull request merged**